website/content/en/configurations/identity-access-modes/system-assigned-msi-mode.md

```diff
@@ -68,9 +68,7 @@ spec:
 
 ## Configure System-assigned Managed Identity to access Keyvault
 
-AKS uses system-assigned managed identity as [cluster managed identity](https://docs.microsoft.com/en-us/azure/aks/use-managed-identity). This managed identity shouldn't be used to access Keyvault. You should consider using a [User-assigned managed identity](./user-assigned-msi-mode) instead.
-
-Before this step, you need to turn on system-assigned managed identity on your cluster VM/VMSS.
+Before this step, you need to [enable system-assigned managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-cli-windows-vm#enable-system-assigned-managed-identity-on-an-existing-azure-vm) in your cluster VM/VMSS.

```

Comment:
> This managed identity shouldn't be used to access Keyvault. You should consider using a [User-assigned managed identity](./user-assigned-msi-mode) instead.
Is this guidance no longer valid?

Reply:
I confirmed this was never the case. There is no system-assigned managed identity created in the cluster. I've added some context in the PR description.

Reply:
Yea I saw the PR description. I guess the key for this PR is to guide users to create system msi on the node and remove any reference about the cluster managed identity right?

Reply:
yeah, that's right. The cluster managed identity is a user-assigned managed identity, so we're stubbing that out of the doc. I've added a doc ref on how to enable system-assigned managed identity. The rest of the doc shows how to verify the system-assigned managed identity exists and role assignments.